### ISSUE TITLE
Screenreader on MacOS and on Ubuntu isn't reading "Monitor Sign Up Promo" and "Firefox Mobile Promo" banners

### ISSUE THREAD

**Issue by sv-hyacoub** [mozilla/blurts-server#1605]
* Affected versions
Chrome Version 80.0.3987.132 (Official Build) (64-bit)
     - Ubuntu and MacOS
Safari Version 13.0.5
     - MacOS
Firefox 74.0 
     - MacOS

* Affected platforms:
MacOS 10.14
Ubuntu

* Preconditions
Download and install a screen reader tool:
Ubuntu: Start "Orca Screen Reader" which is already installed.
Mac: Use "Voice Over" which is already installed. System Preferences> Accesibility> Voice Over> "Enable VoiceOver"

* Steps to reproduce
1. Go to https://blurts-server-review.herokuapp.com
2. Enter any valid email address with breaches and click on "Check for Breaches".
3. Activate the Screen reader.
4. Observe "Monitor Sign Up Promo" banner and "Firefox Mobile Promo" banner after signing in.

* Expected behaviour:
Screenreader on MacOS and on Ubuntu isn't reading "Monitor Sign Up Promo" and "Firefox Mobile Promo" banners.

* Actual behaviour:
Screenreader on MacOS and on Ubuntu should be reading "Monitor Sign Up Promo" and "Firefox Mobile Promo" banners.

**Comment by EMMLynch** [comment]
Closing since we've redesigned the site and functionality since this was created. If you feel that this is still needed, please let me know.
